src/components/Tag/Tag.tsx

```diff
@@ -63,7 +63,7 @@ export const Tag = ({ type, label, onClick, size = TagSize.Medium, 'data-test-id
         <button
             data-test-id={dataTestId}
             className={merge([
-                'tw-inline-flex tw-items-center tw-border tw-border-solid tw-rounded-full tw-text-xs tw-transition-colors tw-group  tw-break-word',
+                'tw-inline-flex tw-items-center tw-rounded-full tw-text-xs tw-transition-colors tw-group  tw-break-word',
```

Comment:
When you click on the tag, the blue outline appears. We should disable it adding that class as well `tw-outline-none`

Reply:
Added